static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx

```diff
@@ -129,13 +147,14 @@ const IconWrapper = styled('div')<Required<Pick<SVGIconProps, 'color'>>>`
   display: flex;
   align-items: center;
   justify-content: center;
-  width: 22px;
-  height: 22px;
+  width: 24px;
+  height: 24px;
   border-radius: 50%;
   color: ${p => p.theme.white};
   background: ${p => p.theme[p.color] ?? p.color};
   box-shadow: ${p => p.theme.dropShadowLightest};
   position: relative;
+  z-index: 2;
```

Comment:
Do we need this set to `2`? How about `p.theme.zIndex.initial`?

It looks like the `2` we have here is meant to win over what's inside the `&::after` selector. But that doesn't need a zindex at all. So `p.theme.zIndex.initial` should be enough here.